docs/framework/unmanaged-api/hosting/icorruntimehost-interface.md

```diff
@@ -71,6 +71,6 @@ Provides methods that enable the host to start and stop the common language runt
  <xref:System.AppDomain>   
  [Hosting](../../../../docs/framework/unmanaged-api/hosting/index.md)   
  [ICLRRuntimeHost Interface](../../../../docs/framework/unmanaged-api/hosting/iclrruntimehost-interface.md)   
- [Runtime Hosts](http://msdn.microsoft.com/en-us/99d9246a-b994-4fe5-985c-8588d1d59998)   
+ [Runtime Hosts](http://msdn.microsoft.com/99d9246a-b994-4fe5-985c-8588d1d59998)   
```

Comment:
@mairaw Yes, this was a "low hanging fruit" PR to just get `en-us` out. Do you want me to rerun the script to ignore the `msdn.microsoft.com` links so that those can be addressed in a future issue/PR? That way, we'd be fixing the more stable/long-term links with my PR?